libgeopm/test/Makefile.mk

```diff
@@ -209,4 +206,13 @@ coverage: init-coverage check
 	lcov --remove coverage-base-combined.info "$$(realpath $$(pwd))/src/geopm_pmpi_fortran.c" --output-file coverage-base-combined-filtered.info
 	genhtml coverage-base-combined-filtered.info --output-directory coverage-base --legend -t $(VERSION) -f
 
+if BUILD_GTEST
 include test/googletest.mk
+test_geopm_test_LDADD += libgtest.a \
+                         libgmock.a \
+                         # end
+else
+test_geopm_test_LDADD += -lgmock \
+                         -lgtest \
```

Comment:
If we're going to try to link against gtest and gmock when BUILD_GTEST is disabled, the documentation should be updated to reflect this new requirement.